src/Symfony/Component/HttpFoundation/Tests/HeaderBagTest.php

```diff
@@ -88,24 +88,37 @@ public function testGet()
         $bag = new HeaderBag(['foo' => 'bar', 'fuzz' => 'bizz']);
         $this->assertEquals('bar', $bag->get('foo'), '->get return current value');
         $this->assertEquals('bar', $bag->get('FoO'), '->get key in case insensitive');
-        $this->assertEquals(['bar'], $bag->get('foo', 'nope', false), '->get return the value as array');
+        $this->assertEquals(['bar'], $bag->all('foo'), '->get return the value as array');
 
         // defaults
         $this->assertNull($bag->get('none'), '->get unknown values returns null');
         $this->assertEquals('default', $bag->get('none', 'default'), '->get unknown values returns default');
-        $this->assertEquals(['default'], $bag->get('none', 'default', false), '->get unknown values returns default as array');
+
+        $this->assertEquals([], $bag->all('none'), '->get unknown values returns an empty array');
 
         $bag->set('foo', 'bor', false);
         $this->assertEquals('bar', $bag->get('foo'), '->get return first value');
-        $this->assertEquals(['bar', 'bor'], $bag->get('foo', 'nope', false), '->get return all values as array');
+
+        $this->assertEquals(['bar', 'bor'], $bag->all('foo'), '->get return all values as array');
+    }
+
+    /**
+     * @group legacy
+     * @expectedDeprecation Passing $first as a third argument is deprecated since Symfony 4.4 and will be removed in Symfony 5.0. Symfony\Component\HttpFoundation\HeaderBag::get will return the first header value, use Symfony\Component\HttpFoundation\HeaderBag::all($key) instead to get an array of values.
```

Comment:
" and will be removed in Symfony 5.0" should be removed, we're trying to get rid of them